app/javascript/react/screens/App/Overview/screens/MappingWizard/components/DualPaneMapper/DualPaneMapper.scss

```diff
@@ -0,0 +1,114 @@
+/** Dual Pane Mapper CSS **/
```

Comment:
much cleaner css naming structure than my CSS from 8 months ago 😸 

👍 

BEM all the things...